be2-scala/src/main/scala/ch/epfl/pop/model/objects/Base64Data.scala

```diff
@@ -0,0 +1,3 @@
+package ch.epfl.pop.model.objects
+
+case class Base64Data(data: String)
```

Comment:
What is the difference between `Base64Data` and the `Base64String` used in `MessageCreationUtils`?

Reply:
Same idea. Base64String is the old Base64Data. But since we didn't touch the data part yet, it is not yet refactored :)

Reply:
Ok great, got a bit confused there

Reply:
@celinecamacho One more thing: the only difference is that Base64String was a type alias whereas Base64Data is a case class. That means that we can add methods such as decode directly in the base64data class now :)